stats.c

```diff
@@ -770,8 +771,9 @@ static void collect_barcode_stats(bam1_t* bam_line, stats_t* stats) {
                 break;
             default:
                 if (*separator >= 0) {
-                    if (*separator != i) {
-                        fprintf(stderr, "Barcode separator for tag %s is in a different position at sequence '%s'\n", barcode_tag, bam_get_qname(bam_line));
+                    if (*separator != i && !error_raised) {
```

Comment:
This will fail to `continue` if the error has already been raised.  I think it needs to be:
```
if (*separator != i) {
    if (!error_raised) { ... }
    continue;
}
```

Reply:
Thanks for spotting that! `continue` there was useless, anyway. 
The question now is whether we still want to process the rest of the barcode (and its associated quality tag) or  skip to the next tag.